elixir/apps/api/lib/api/gateway/views/client.ex

```diff
@@ -2,12 +2,32 @@ defmodule API.Gateway.Views.Client do
   alias Domain.Clients
 
   def render(%Clients.Client{} = client, preshared_key) do
+    # The OS name can have spaces, hence split the user-agent step by step.
+    [os_name, rest] = String.split(client.last_seen_user_agent, "/", parts: 2)
+    [os_version, rest] = String.split(rest, " ", parts: 2)
+    [_, rest] = String.split(rest, "/", parts: 2)
+
+    # FIXME: For easier testing, we re-parse the client version here.
```

Comment:
FIXMEs are prohibited by the linter.

```suggestion
    # TODO: For easier testing, we re-parse the client version here.
```